code/game/objects/structures/crates_lockers/closets/fitness.dm

```diff
@@ -11,6 +11,8 @@
 	new /obj/item/clothing/under/shorts/red(src)
 	new /obj/item/clothing/under/shorts/blue(src)
 	new /obj/item/clothing/under/shorts/green(src)
+	for(var/i in 1 to 4)
+		new /obj/item/clothing/ears/headphones(src)

```

Comment:
They really should only spawn in one place instead of 3 different locations imo.